gen/gcli/gcli_test.go

```diff
@@ -12,7 +13,7 @@ import (
 )
 
 type cfg1 struct {
-	StringValue1 string
+	StringValue1 string `flag:",required"`
 	StringValue2 string `flag:"string-value-two s"`
```

Comment:
Why string with no specifiers is removed from the test?

Reply:
Reverted and new field added